<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job Accounting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/JobAccounting.png</normaloff>:/JobAccounting.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
			    background-color: rgb(255, 255, 255);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
              }
QMenuBar {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QStatusBar {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenu {
                background-color: #fff;
                border: none;
                border-radius: 0px;
                }
                QMenu::item {
                padding-left: 18px;
                margin: 4px;
                }
                QMenu::item:selected {
                color: #111;
                background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                        stop: 0 #fff, stop: 1 #eef);
                }
                QMenu::separator {
                height: .5px;
                background: #eee;
                margin-left: 10px;
                margin-right: 5px;
                }
                QMenu::indicator {
                width: 14px;
                height: 14px;
                }
                QMenu::indicator:non-exclusive:checked {
                image: url(:/pict/check.png);
                }
                QMenu::indicator:non-exclusive:unchecked {
                image: url(:/pict/unchecked.png);
           }</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuJob_accounting">
    <property name="title">
     <string>Accounting</string>
    </property>
    <addaction name="actionJob_accounting"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuAdministration">
    <property name="title">
     <string>Administration</string>
    </property>
    <addaction name="actionUsers"/>
    <addaction name="actionWorkers"/>
    <addaction name="actionEquipment"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
    <addaction name="actionUkrainian"/>
   </widget>
   <addaction name="menuJob_accounting"/>
   <addaction name="menuAdministration"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionUsers">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/user.png</normaloff>:/user.png</iconset>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionWorkers">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/worker.png</normaloff>:/worker.png</iconset>
   </property>
   <property name="text">
    <string>Workers</string>
   </property>
  </action>
  <action name="actionEquipment">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/equipment.png</normaloff>:/equipment.png</iconset>
   </property>
   <property name="text">
    <string>Equipment</string>
   </property>
  </action>
  <action name="actionJob_accounting">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/calculator.png</normaloff>:/calculator.png</iconset>
   </property>
   <property name="text">
    <string>Job accounting</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/english.png</normaloff>:/english.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/russian.png</normaloff>:/russian.png</iconset>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionUkrainian">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/ukrainian.png</normaloff>:/ukrainian.png</iconset>
   </property>
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/log.png</normaloff>:/log.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
